### PR TITLE
Add c++ counterparts for java shuffleboard snippets

### PR DESCRIPTION
**Pull request by cfinn68916**

### CODE REVIEW

File: source/docs/software/dashboards/shuffleboard/layouts-with-code/sending-data.rst [hunk 1]
```diff
@@ -52,6 +52,11 @@ Simply using `addPersistent` instead of `add` will make the value saved on the r
        Shuffleboard.getTab("Drive")
             .addPersistent("Max Speed", 1.0);
 
+   .. code-tab:: c++
+
+       Shuffleboard::GetTab("Tab Title")
+            .AddPersistent("Sendable Title", mySendable);
```
Comment (Starlight220) [inline source/docs/software/dashboards/shuffleboard/layouts-with-code/sending-data.rst:58]: These strings should match the ones in the Java snippet.

**Review by Starlight220 — CHANGES_REQUESTED** [review]

File: source/docs/software/dashboards/shuffleboard/layouts-with-code/sending-data.rst [hunk 2]
```diff
@@ -63,3 +68,8 @@ Analogous to ``SmartDashboard.putData``, any ``Sendable`` object (most sensors,
 
        Shuffleboard.getTab("Tab Title")
             .add("Sendable Title", mySendable);
+
+   .. code-tab:: c++
+
+       Shuffleboard::GetTab("Tab Title")
+            .add("Sendable Title", mySendable);
```
Comment (calcmogul) [inline source/docs/software/dashboards/shuffleboard/layouts-with-code/sending-data.rst:75]:
```suggestion
            .Add("Sendable Title", mySendable);
```

**Review by calcmogul — CHANGES_REQUESTED** [review]

**Review by Starlight220 — COMMENTED** [review]
Should we namespace-qualify here? What do we do usually?

**Comment by calcmogul** [comment]
The WPILib examples include the `frc::` prefix.

**Comment by Starlight220** [comment]
So yeah, stuff here should be qualified with the `frc::` prefix.

**Comment by cfinn68916** [comment]
Can you sqaush the PR when you merge?

**Comment by sciencewhiz** [comment]
Don't worry, we squash by default, unless there's a reason not to.

**Review by sciencewhiz — APPROVED** [review]

**Pull request merged**